miden/src/cli/verify.rs

```diff
@@ -41,7 +41,7 @@ impl VerifyCmd {
         // load proof from file
         let proof = ProofFile::read(&Some(self.proof_file.clone()), &self.proof_file)?;
 
-        println!("verifying program...");
+        println!("Verifying program...");
```

Comment:
I'm wondering if we need to print out this line. How does the output for this command look like?

Reply:
I think we should remove it.